manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -70,9 +70,6 @@ spec:
             - name: kubelet-dir
               mountPath: /var/lib/kubelet
               mountPropagation: "Bidirectional"
-            - name: pods-cloud-data
-              mountPath: /var/lib/cloud/data
```

Comment:
@kayrus IMO, If its a valid and supported (not deprecated) way of retrieving data (eg: instanceid as mentioned in issue) . Removing is not a solution. I suggest we add a search order instead where we give user capability to choose from where he wants to retrieve the data . If its not really used (I dont have idea on that, others can chime in) , how about we deprecate this eventually and later be removed , documenting this way is not supported.

Reply:
/var/lib/cloud/data seems to be a cloud-init thing? 
and agree that remove this seems too rush for now, maybe add some warning first. and if
someone has the /var/lib/cloud/data before, this will be a problem for them now

Reply:
agree, removed.